test/unit-test/cellular_at_core_utest.c

```diff
@@ -1005,9 +1004,9 @@ void test_Cellular_ATIsPrefixPresent_Happy_Path( void )
 {
     CellularATError_t cellularStatus = CELLULAR_AT_SUCCESS;
     char * pStr = malloc( sizeof( char ) * ( strlen( CELLULAR_SAMPLE_PREFIX_STRING_INPUT ) + 1 ) );
```

Comment:
Perhaps if you want it allocated and initialized:
char  pStr[CELLULAR_SAMPLE_PREFIX_STRING_INPUT] = {0};

Reply:
That sounds good, thanks for the suggestion.

Reply:
please refer to this [change](https://github.com/FreeRTOS/Lab-Project-FreeRTOS-Cellular-Library/pull/37/commits/d6ed9baaca207edc8cb5ab7fe890e2937975d2d8).